### PR TITLE
fixed succes error + renamed tags to extra tags + make db creation att…

### PR DESCRIPTION
**Pull request by pjcalvo**
This PR tackles 3 issues:

1) Renamed tags for additional to make a separation of what we internally use vs what the users is sending us
2) Fixed a problem where `if response:` was never resolving and therefore all requests where success.
3) Attempt to connect and create a influxdb every time instead of using a parameter for it. 

If this PR is approved it will become our first 1.0 release... 

NOTE: There are breaking changes on the names of the host and port parameters.

### CODE REVIEW

**Comment by pjcalvo** [comment]
@jmfiola Hey my friend. I have invited you to the repository, so you can also merge the PRs for me. 

On this PR I changed the tags property that you added for additional_tags so that we can make a split between what we manage internally and what a customer sends us. If you look at this screenshot you can see that it still works.

image

Also, I made an additional changes to fix a bug and improve the database creation. I know that I should have done separates PRs apologies for that.

Finally, this will be our 1.0 release since it is stable and seems to have proven that it works.

File: locust_influxdb_listener/__init__.py [hunk 1]
```diff
@@ -6,40 +8,52 @@
...
+
+        :param host: InfluxDB host address or hostname.
+        :param port: InfluxDB HTTP API port.
+        :param user: InfluxDB username for authentication.
+        :param pwd: InfluxDB password for authentication.
+        :param database: InfluxDB database name for storing data.
+        :param interval_ms: Data sending interval in milliseconds.
+        :param ssl: Enable SSL/TLS for secure data transmission.
+        :param verify_ssl: Verify SSL certificates (only if SSL is enabled).
+        :param additional_tags: Additional tags to include in global data points.
```
Comment (jmfiola) [inline locust_influxdb_listener/__init__.py]: would reword this: Additional tags to include in globally for all data points*

**Review by jmfiola — APPROVED** [review]
LGTM! Excited for the 1.0 release and automate versionin bumping + publishing! Thanks

**Comment by jmfiola** [comment]
Sorry for just now seeing this! I need to update my e-mail notifications for github I think.

**Comment by jmfiola** [comment]
Oh also you may want to bump the version to 1.0

File: README.md [hunk 2]
```diff
@@ -32,15 +32,16 @@ def on_locust_init(environment, **_kwargs):
     """
     # this settings matches the given docker-compose file
     influxDBSettings = InfluxDBSettings(
-        influx_host = 'localhost',
-        influx_port = '8086',
+        host = 'localhost',
```
Comment (jmfiola) [inline README.md:35]: Ah I know I already approved, but going to revoke it, as I would prefer if we didn't change these param names, as we have some locust-grasshopper kwargs that are using `influx_host` and `influx_port` and passing it in like this:

```
influx_db_settings = InfluxDBSettings(
                **influx_configuration,
                database="locust",
            )
```

So if you made this change then we would have to go and change it in a lot of places :(

**Review by jmfiola — CHANGES_REQUESTED** [review]

**Comment by pjcalvo** [comment]
I am going to ads both params for now :) no problem

**Comment by pjcalvo** [comment]
> Oh also you may want to bump the version to 1.0

Yeah but that would come on the release commit

File: README.md (hunk 2 above)
Comment (pjcalvo) [reply README.md:35]: OK I left both for now. And added unit tests for it
